Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/Migration/Tools/MigrationWindow.cs

```diff
@@ -0,0 +1,417 @@
+                    {
+                        GUILayout.FlexibleSpace();
+
+                        string tooltip = "All Scenes and Prefabs Selected";
+                        EditorGUILayout.LabelField(new GUIContent(tooltip, InspectorUIUtility.WarningIcon));
+                    }
+                }
+            }
+        }
+
+        private void DrawMigrationObjectsList()
+        {
+            scrollPosition = EditorGUILayout.BeginScrollView(scrollPosition);
+
+            foreach (var migrationObject in migrationObjects)
+            {
+                using (var horizontal = new GUILayout.HorizontalScope())
+                {
+                    using (new EditorGUI.DisabledGroupScope(true))
+                    {
+                        EditorGUILayout.ObjectField(migrationObject, typeof(Object), false);
+                    }
+                    var icon = EditorGUIUtility.IconContent("winbtn_win_min_h");
+
+                    if (GUILayout.Button(icon, GUILayout.Width(30)))
+                    {
+                        migrationObjects.Remove(migrationObject);
+                        break;
+                    }
+                }
+            }
+            EditorGUILayout.Space();
+
+            using (new GUILayout.VerticalScope(EditorStyles.helpBox))
+            {
+                if (migrationObjects.Count == 0)
+                {
+                    GUI.enabled = false;
+                }
+
+                if (GUILayout.Button("Migrate"))
+                {
+                    MigrateSelection();
+                }
+
+                using (new EditorGUILayout.HorizontalScope())
+                {
+                    if (migrationObjects.Count > 0)
+                    {
+                        GUILayout.FlexibleSpace();
+
+                        string tooltip = $"{migrationObjects.Count} {toolbarTitles[(int)selectedToolbar]} selected for migration";
+                        EditorGUILayout.LabelField(new GUIContent(tooltip, InspectorUIUtility.WarningIcon));
+                    }
+                }
+            }
+            EditorGUILayout.EndScrollView();
+        }
+
+        private void AddSelection()
+        {
+            if (selectedToolbar == ToolbarOption.Objects)
+            {
+                if (!migrationObjects.Contains(selection))
+                {
+                    migrationObjects.Add(selection);
+                }
+            }
+            else
+            {
+                if (!migrationObjects.Contains(selection) && IsExtension(AssetDatabase.GetAssetPath(selection), SceneExtension))
+                {
+                    migrationObjects.Add(selection);
+                }
+            }
+        }
+
+        private void MigrateAllScenes()
+        {
+            EditorSceneManager.SaveCurrentModifiedScenesIfUserWantsTo();
+                 
+            var previousScenePath = EditorSceneManager.GetActiveScene().path;
```

Comment:
I think this variable is unused here right? I think it may have been copy pasta from MigrateScene